components/mpas-ocean/src/shared/Makefile

```diff
@@ -92,7 +92,7 @@ mpas_ocn_thick_vadv.o: mpas_ocn_constants.o mpas_ocn_config.o
 
 mpas_ocn_thick_surface_flux.o: mpas_ocn_forcing.o mpas_ocn_constants.o mpas_ocn_config.o
 
-mpas_ocn_gm.o:  mpas_ocn_constants.o mpas_ocn_config.o
+mpas_ocn_gm.o:  mpas_ocn_constants.o mpas_ocn_config.o mpas_ocn_diagnostics_variables.o
```

Comment:
Same question here

Reply:
I just pushed a new commit that reverts this Makefile change.